lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers_test.go

```diff
@@ -666,6 +666,222 @@ func TestGetSubscriber(t *testing.T) {
 	tests.RunUnitTest(t, e, tc)
 }
 
+func TestListSubscriberStates(t *testing.T) {
```

Comment:
These two test functions have a lot of shared code to test relatively similar endpoints. I'd consider just combining them or pull out helper functions.

Reply:
Agreed. The shared code though is mostly by convenience rather than requirement, and this test is a lot shorter, so I'll just leave it for now